src/ert/parsing/config_keywords.py

```diff
@@ -226,45 +259,80 @@ def convert(
-            return path
+                raise ConfigValidationError.from_info(
+                    ErrorInfo(message=err, filename=token.filename).set_context(token)
+                )
+            return ContextString(path, token, keyword)
         if val_type == SchemaType.CONFIG_EXECUTABLE:
             path = str(token)
             if not os.path.isabs(token) and not os.path.exists(token):
                 path = shutil.which(token)
-                if path is None:
-                    raise ConfigValidationError(
-                        f"Could not find executable {token.value!r}",
-                        config_file=token.filename,
-                    )
+
+            if path is None:
+                raise ConfigValidationError.from_info(
+                    ErrorInfo(
+                        message=f"Could not find executable {token.value!r}",
+                        filename=token.filename,
+                    ).set_context(token)
+                )
+
             if not os.access(path, os.X_OK):
                 context = (
                     f"{token.value!r} which was resolved to {path!r}"
                     if token.value != path
                     else f"{token.value!r}"
                 )
-                raise ConfigValidationError(
-                    f"File not executable: {context}",
-                    config_file=token.filename,
+                raise ConfigValidationError.from_info(
+                    ErrorInfo(
+                        message=f"File not executable: {context}",
+                        filename=token.filename,
+                    ).set_context(token)
                 )
-            return path
-        return str(token)
-
-    def apply_constraints(self, args: List[Any]) -> Union[List[Any], Any]:
-        args = [
-            self.convert(x, i) if isinstance(x, FileContextToken) else x
-            for i, x in enumerate(args)
-        ]
+            return ContextString(path, token, keyword)
+        return ContextString(str(token), token, keyword)
+
+    def apply_constraints(
+        self,
+        args: List[Any],
+        keyword: FileContextToken,
+    ) -> Union[List[Any], Any]:
+        errors = []
+
+        args_with_context = []
+        for i, x in enumerate(args):
+            if isinstance(x, FileContextToken):
+                try:
+                    value_with_context = self.token_to_value_with_context(x, i, keyword)
+                    args_with_context.append(value_with_context)
+                except ConfigValidationError as err:
+                    errors.append(err)
+                    continue
+            else:
+                args_with_context.append(x)
+
+        args = args_with_context
+
         if self.argc_min != -1 and len(args) < self.argc_min:
-            raise ConfigValidationError(
-                f"{self.kw} must have at least {self.argc_min} arguments"
+            errors.append(
+                ErrorInfo(
```

Comment:
mypy needs a hint for the type of the errors list here.

Reply:
Fixed